src/components/SecondaryPanes/Frames/Frame.js

```diff
@@ -118,12 +114,20 @@ export default class FrameComponent extends Component<FrameComponentProps> {
       selectedFrame,
       hideLocation,
       shouldMapDisplayName,
-      displayFullUrl
+      displayFullUrl,
+      getFrameTitle
     } = this.props;
 
     const className = classNames("frame", {
       selected: selectedFrame && selectedFrame.id === frame.id
     });
+
+    const title = getFrameTitle
+      ? getFrameTitle(
+          `${getFileURL(frame.source, false)}:${frame.location.line}`
+        )
+      : undefined;
```

Comment:
Would `""` be better here?

Reply:
`""` would create an empty `title` attribute in the dom node, whereas `undefined` do not.
So, that's not a really big deal, but I'd rather not render anything on the DOM that is not useful.